<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SecondWindow</class>
 <widget class="QMainWindow" name="SecondWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>20</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Game Level</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btu2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>390</y>
      <width>131</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft PhagsPa</family>
      <pointsize>20</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget {
    /* 分层背景：图片在下，渐变在上 */
    background-image: url(:/res/ground.jpg),
                     qradialgradient(spread:repeat,
                                    cx:0.5, cy:0.5, radius:0.077,
                                    fx:0.5, fy:0.5,
                                    stop:0 rgba(0, 169, 255, 58),
                                    stop:0.5 rgba(0, 0, 0, 58),
                                    stop:1 rgba(0, 169, 255, 58));
    
    /* 背景混合模式 */
    background-blend-mode: overlay;
    
    /* 备用背景色 */
    background-color: rgb(85, 170, 127);
    
    /* 字体设置 */
    font: 20pt &quot;Microsoft PhagsPa&quot;;
}</string>
    </property>
    <property name="text">
     <string>读取地图</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btu1">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>390</y>
      <width>121</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft PhagsPa</family>
      <pointsize>20</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget {
    /* 分层背景：图片在下，渐变在上 */
    background-image: url(:/res/ground.jpg),
                     qradialgradient(spread:repeat,
                                    cx:0.5, cy:0.5, radius:0.077,
                                    fx:0.5, fy:0.5,
                                    stop:0 rgba(0, 169, 255, 58),
                                    stop:0.5 rgba(0, 0, 0, 58),
                                    stop:1 rgba(0, 169, 255, 58));
    
    /* 背景混合模式 */
    background-blend-mode: overlay;
    
    /* 备用背景色 */
    background-color: rgb(85, 170, 127);
    
    /* 字体设置 */
    font: 20pt &quot;Microsoft PhagsPa&quot;;
}</string>
    </property>
    <property name="text">
     <string>返回主页</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btu3">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>111</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft PhagsPa</family>
      <pointsize>18</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget {
    /* 分层背景：图片在下，渐变在上 */
    background-image: url(:/res/1.jpg),
                     qradialgradient(spread:repeat,
                                    cx:0.5, cy:0.5, radius:0.077,
                                    fx:0.5, fy:0.5,
                                    stop:0 rgba(0, 169, 255, 58),
                                    stop:0.5 rgba(0, 0, 0, 58),
                                    stop:1 rgba(0, 169, 255, 58));
    
    /* 背景混合模式 */
    background-blend-mode: overlay;
    
    /* 备用背景色 */
    background-color: rgb(85, 170, 127);
    
    /* 字体设置 */
    font: 18pt &quot;Microsoft PhagsPa&quot;;
}</string>
    </property>
    <property name="text">
     <string>第一关</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btu5">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>50</y>
      <width>111</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft PhagsPa</family>
      <pointsize>18</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget {
    /* 分层背景：图片在下，渐变在上 */
    background-image: url(:/res/1.jpg),
                     qradialgradient(spread:repeat,
                                    cx:0.5, cy:0.5, radius:0.077,
                                    fx:0.5, fy:0.5,
                                    stop:0 rgba(0, 169, 255, 58),
                                    stop:0.5 rgba(0, 0, 0, 58),
                                    stop:1 rgba(0, 169, 255, 58));
    
    /* 背景混合模式 */
    background-blend-mode: overlay;
    
    /* 备用背景色 */
    background-color: rgb(85, 170, 127);
    
    /* 字体设置 */
    font: 18pt &quot;Microsoft PhagsPa&quot;;
}</string>
    </property>
    <property name="text">
     <string>第二关</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btu8">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>140</y>
      <width>111</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft PhagsPa</family>
      <pointsize>18</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget {
    /* 分层背景：图片在下，渐变在上 */
    background-image: url(:/res/1.jpg),
                     qradialgradient(spread:repeat,
                                    cx:0.5, cy:0.5, radius:0.077,
                                    fx:0.5, fy:0.5,
                                    stop:0 rgba(0, 169, 255, 58),
                                    stop:0.5 rgba(0, 0, 0, 58),
                                    stop:1 rgba(0, 169, 255, 58));
    
    /* 背景混合模式 */
    background-blend-mode: overlay;
    
    /* 备用背景色 */
    background-color: rgb(85, 170, 127);
    
    /* 字体设置 */
    font: 18pt &quot;Microsoft PhagsPa&quot;;
}</string>
    </property>
    <property name="text">
     <string>第五关</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btu9">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>140</y>
      <width>111</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft PhagsPa</family>
      <pointsize>18</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget {
    /* 分层背景：图片在下，渐变在上 */
    background-image: url(:/res/1.jpg),
                     qradialgradient(spread:repeat,
                                    cx:0.5, cy:0.5, radius:0.077,
                                    fx:0.5, fy:0.5,
                                    stop:0 rgba(0, 169, 255, 58),
                                    stop:0.5 rgba(0, 0, 0, 58),
                                    stop:1 rgba(0, 169, 255, 58));
    
    /* 背景混合模式 */
    background-blend-mode: overlay;
    
    /* 备用背景色 */
    background-color: rgb(85, 170, 127);
    
    /* 字体设置 */
    font: 18pt &quot;Microsoft PhagsPa&quot;;
}</string>
    </property>
    <property name="text">
     <string>第六关</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btu6">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>50</y>
      <width>111</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft PhagsPa</family>
      <pointsize>18</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget {
    /* 分层背景：图片在下，渐变在上 */
    background-image: url(:/res/1.jpg),
                     qradialgradient(spread:repeat,
                                    cx:0.5, cy:0.5, radius:0.077,
                                    fx:0.5, fy:0.5,
                                    stop:0 rgba(0, 169, 255, 58),
                                    stop:0.5 rgba(0, 0, 0, 58),
                                    stop:1 rgba(0, 169, 255, 58));
    
    /* 背景混合模式 */
    background-blend-mode: overlay;
    
    /* 备用背景色 */
    background-color: rgb(85, 170, 127);
    
    /* 字体设置 */
    font: 18pt &quot;Microsoft PhagsPa&quot;;
}</string>
    </property>
    <property name="text">
     <string>第三关</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btu10">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>140</y>
      <width>111</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft PhagsPa</family>
      <pointsize>18</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget {
    /* 分层背景：图片在下，渐变在上 */
    background-image: url(:/res/1.jpg),
                     qradialgradient(spread:repeat,
                                    cx:0.5, cy:0.5, radius:0.077,
                                    fx:0.5, fy:0.5,
                                    stop:0 rgba(0, 169, 255, 58),
                                    stop:0.5 rgba(0, 0, 0, 58),
                                    stop:1 rgba(0, 169, 255, 58));
    
    /* 背景混合模式 */
    background-blend-mode: overlay;
    
    /* 备用背景色 */
    background-color: rgb(85, 170, 127);
    
    /* 字体设置 */
    font: 18pt &quot;Microsoft PhagsPa&quot;;
}</string>
    </property>
    <property name="text">
     <string>第七关</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btu7">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>50</y>
      <width>121</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft PhagsPa</family>
      <pointsize>18</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget {
    /* 分层背景：图片在下，渐变在上 */
    background-image: url(:/res/1.jpg),
                     qradialgradient(spread:repeat,
                                    cx:0.5, cy:0.5, radius:0.077,
                                    fx:0.5, fy:0.5,
                                    stop:0 rgba(0, 169, 255, 58),
                                    stop:0.5 rgba(0, 0, 0, 58),
                                    stop:1 rgba(0, 169, 255, 58));
    
    /* 背景混合模式 */
    background-blend-mode: overlay;
    
    /* 备用背景色 */
    background-color: rgb(85, 170, 127);
    
    /* 字体设置 */
    font: 18pt &quot;Microsoft PhagsPa&quot;;
}</string>
    </property>
    <property name="text">
     <string>第四关</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btu11">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>140</y>
      <width>121</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft PhagsPa</family>
      <pointsize>18</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget {
    /* 分层背景：图片在下，渐变在上 */
    background-image: url(:/res/1.jpg),
                     qradialgradient(spread:repeat,
                                    cx:0.5, cy:0.5, radius:0.077,
                                    fx:0.5, fy:0.5,
                                    stop:0 rgba(0, 169, 255, 58),
                                    stop:0.5 rgba(0, 0, 0, 58),
                                    stop:1 rgba(0, 169, 255, 58));
    
    /* 背景混合模式 */
    background-blend-mode: overlay;
    
    /* 备用背景色 */
    background-color: rgb(85, 170, 127);
    
    /* 字体设置 */
    font: 18pt &quot;Microsoft PhagsPa&quot;;
}</string>
    </property>
    <property name="text">
     <string>第八关</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btu12">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>220</y>
      <width>111</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft PhagsPa</family>
      <pointsize>18</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget {
    /* 分层背景：图片在下，渐变在上 */
    background-image: url(:/res/1.jpg),
                     qradialgradient(spread:repeat,
                                    cx:0.5, cy:0.5, radius:0.077,
                                    fx:0.5, fy:0.5,
                                    stop:0 rgba(0, 169, 255, 58),
                                    stop:0.5 rgba(0, 0, 0, 58),
                                    stop:1 rgba(0, 169, 255, 58));
    
    /* 背景混合模式 */
    background-blend-mode: overlay;
    
    /* 备用背景色 */
    background-color: rgb(85, 170, 127);
    
    /* 字体设置 */
    font: 18pt &quot;Microsoft PhagsPa&quot;;
}</string>
    </property>
    <property name="text">
     <string>第九关</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btu16">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>300</y>
      <width>111</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft PhagsPa</family>
      <pointsize>18</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget {
    /* 分层背景：图片在下，渐变在上 */
    background-image: url(:/res/1.jpg),
                     qradialgradient(spread:repeat,
                                    cx:0.5, cy:0.5, radius:0.077,
                                    fx:0.5, fy:0.5,
                                    stop:0 rgba(0, 169, 255, 58),
                                    stop:0.5 rgba(0, 0, 0, 58),
                                    stop:1 rgba(0, 169, 255, 58));
    
    /* 背景混合模式 */
    background-blend-mode: overlay;
    
    /* 备用背景色 */
    background-color: rgb(85, 170, 127);
    
    /* 字体设置 */
    font: 18pt &quot;Microsoft PhagsPa&quot;;
}</string>
    </property>
    <property name="text">
     <string>第十三关</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btu13">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>220</y>
      <width>111</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft PhagsPa</family>
      <pointsize>18</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget {
    /* 分层背景：图片在下，渐变在上 */
    background-image: url(:/res/1.jpg),
                     qradialgradient(spread:repeat,
                                    cx:0.5, cy:0.5, radius:0.077,
                                    fx:0.5, fy:0.5,
                                    stop:0 rgba(0, 169, 255, 58),
                                    stop:0.5 rgba(0, 0, 0, 58),
                                    stop:1 rgba(0, 169, 255, 58));
    
    /* 背景混合模式 */
    background-blend-mode: overlay;
    
    /* 备用背景色 */
    background-color: rgb(85, 170, 127);
    
    /* 字体设置 */
    font: 18pt &quot;Microsoft PhagsPa&quot;;
}</string>
    </property>
    <property name="text">
     <string>第十个</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btu19">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>300</y>
      <width>121</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft PhagsPa</family>
      <pointsize>18</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget {
    /* 分层背景：图片在下，渐变在上 */
    background-image: url(:/res/1.jpg),
                     qradialgradient(spread:repeat,
                                    cx:0.5, cy:0.5, radius:0.077,
                                    fx:0.5, fy:0.5,
                                    stop:0 rgba(0, 169, 255, 58),
                                    stop:0.5 rgba(0, 0, 0, 58),
                                    stop:1 rgba(0, 169, 255, 58));
    
    /* 背景混合模式 */
    background-blend-mode: overlay;
    
    /* 备用背景色 */
    background-color: rgb(85, 170, 127);
    
    /* 字体设置 */
    font: 18pt &quot;Microsoft PhagsPa&quot;;
}</string>
    </property>
    <property name="text">
     <string>第十六关</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btu14">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>220</y>
      <width>111</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft PhagsPa</family>
      <pointsize>18</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget {
    /* 分层背景：图片在下，渐变在上 */
    background-image: url(:/res/1.jpg),
                     qradialgradient(spread:repeat,
                                    cx:0.5, cy:0.5, radius:0.077,
                                    fx:0.5, fy:0.5,
                                    stop:0 rgba(0, 169, 255, 58),
                                    stop:0.5 rgba(0, 0, 0, 58),
                                    stop:1 rgba(0, 169, 255, 58));
    
    /* 背景混合模式 */
    background-blend-mode: overlay;
    
    /* 备用背景色 */
    background-color: rgb(85, 170, 127);
    
    /* 字体设置 */
    font: 18pt &quot;Microsoft PhagsPa&quot;;
}</string>
    </property>
    <property name="text">
     <string>第十一关</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btu18">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>300</y>
      <width>111</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft PhagsPa</family>
      <pointsize>18</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget {
    /* 分层背景：图片在下，渐变在上 */
    background-image: url(:/res/1.jpg),
                     qradialgradient(spread:repeat,
                                    cx:0.5, cy:0.5, radius:0.077,
                                    fx:0.5, fy:0.5,
                                    stop:0 rgba(0, 169, 255, 58),
                                    stop:0.5 rgba(0, 0, 0, 58),
                                    stop:1 rgba(0, 169, 255, 58));
    
    /* 背景混合模式 */
    background-blend-mode: overlay;
    
    /* 备用背景色 */
    background-color: rgb(85, 170, 127);
    
    /* 字体设置 */
    font: 18pt &quot;Microsoft PhagsPa&quot;;
}</string>
    </property>
    <property name="text">
     <string>第十五关</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btu15">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>220</y>
      <width>121</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft PhagsPa</family>
      <pointsize>18</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget {
    /* 分层背景：图片在下，渐变在上 */
    background-image: url(:/res/1.jpg),
                     qradialgradient(spread:repeat,
                                    cx:0.5, cy:0.5, radius:0.077,
                                    fx:0.5, fy:0.5,
                                    stop:0 rgba(0, 169, 255, 58),
                                    stop:0.5 rgba(0, 0, 0, 58),
                                    stop:1 rgba(0, 169, 255, 58));
    
    /* 背景混合模式 */
    background-blend-mode: overlay;
    
    /* 备用背景色 */
    background-color: rgb(85, 170, 127);
    
    /* 字体设置 */
    font: 18pt &quot;Microsoft PhagsPa&quot;;
}</string>
    </property>
    <property name="text">
     <string>第十二关</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btu17">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>300</y>
      <width>111</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft PhagsPa</family>
      <pointsize>18</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget {
    /* 分层背景：图片在下，渐变在上 */
    background-image: url(:/res/1.jpg),
                     qradialgradient(spread:repeat,
                                    cx:0.5, cy:0.5, radius:0.077,
                                    fx:0.5, fy:0.5,
                                    stop:0 rgba(0, 169, 255, 58),
                                    stop:0.5 rgba(0, 0, 0, 58),
                                    stop:1 rgba(0, 169, 255, 58));
    
    /* 背景混合模式 */
    background-blend-mode: overlay;
    
    /* 备用背景色 */
    background-color: rgb(85, 170, 127);
    
    /* 字体设置 */
    font: 18pt &quot;Microsoft PhagsPa&quot;;
}</string>
    </property>
    <property name="text">
     <string>第十四关</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
